### PR TITLE
improve coverage

### PR DESCRIPTION
**Pull request by Khabib73**
Improve coverage to 100% for `PartialRestoreFilter`. Cover the lines 39, 64

## Summary by Sourcery

Tests:
- Add unit test coverage for empty partial-restore pattern handling and database-pattern matching.

### CODE REVIEW

File: tests/unit/test_partial_restore_matcher.py [hunk 1]
```diff
@@ -30,3 +30,24 @@ def test_contains_table(patterns, db_name, table_name, included_result):
...
+    ),
+)
+def test_is_empty_patterns(patterns, db_name, table_name, included_result):
+    pattern_matcher = PartialRestoreFilter(patterns=patterns, inverted=False)
+    assert pattern_matcher.accept_table(db_name, table_name) == included_result
+
+
+@pytest.mark.parametrize(
+    ("database", "pattern_str", "db", "included_result"),
+    (["db1", "db1.table1", "db1", True], ["db2", "db2.table2", "some_db", False]),
```
[the remark this answers is not in the document]
Comment (Khabib73) [reply tests/unit/test_partial_restore_matcher.py]: done

**Review by Alex-Burmak — APPROVED** [review]

**Comment by Khabib73** [comment]
Thanks!
